pages/index.js

```diff
@@ -42,4 +44,52 @@ Home.defaultProps = {
         'Dr. Jenny Liu’s work is featured at the Society of Clinical Psychology. A small excerpt from her article is quoted below...',
     },
   },
+  sponsors: [
+    {
+      img: '/images/sponsors/Ryerson.jpg',
+      alt: 'Ryerson University',
+    },
+    {
+      img: '/images/sponsors/RBC.png',
+      alt: 'Royal Bank of Canada',
+    },
+    {
+      img: '/images/sponsors/CPA.jpg',
+      alt: 'Canadian Psychological Association',
+    },
+    {
+      img: '/images/sponsors/Mitacs.jpg',
+      alt: 'Mitacs',
+    },
+    {
+      img: '/images/sponsors/PFC-300x300.png',
+      alt: 'The Psychology Foundation of Canada',
+    },
+    {
+      img:
+        '/images/sponsors/1200px-American_Psychological_Association_logo.png',
+      alt: 'American_Psychological_Association_logo',
+    },
+    {
+      img: '/images/sponsors/cihr-logo-300x168.jpg',
+      alt: 'Canadian Institutes of Health Research',
+    },
+    {
+      img: '/images/sponsors/download-300x85.jpg',
+      alt: 'Global Alliance for Chronic Diseases',
+    },
+    {
+      img: '/images/sponsors/FIT_ID_Eng_Horiz_CMYK-300x106.jpg',
+      alt: 'Fund for Innovation and Transformation',
+    },
+    {
+      img: '/images/sponsors/foundation-logo-1-300x236.png',
+      alt: 'Hong Fook Mental Health Foundation',
+    },
+    {
+      img:
+        '/images/sponsors/Social_Sciences_and_Humanities_Research_Council_of_Canada_Govern-300x120.jpg',
+      alt: 'New Frontiers in Research Fund',
+    },
+  ],
 }
```

Comment:
Is the formatting correct here? it looks messed up on my screen
![image](https://user-images.githubusercontent.com/65472533/108618778-d0f41580-73ee-11eb-93cc-9199c39f12e3.png)